tests/data/greek-verb-paradigm.test.js

```diff
@@ -0,0 +1,856 @@
+import { Constants, Feature, LanguageModelFactory } from 'alpheios-data-models'
+
+import ViewSetFactory from '@views/lib/view-set-factory.js'
+// import LanguageDatasetFactory from '@views/lib/language-dataset-factory.js'
+
+describe('greek-verb-paradigm.test.js', () => {
+  console.error = function () {}
+  console.log = function () {}
+  console.warn = function () {}
+
+  const locale = "en-US"
+  beforeAll(async () => {
+  })
+
+  beforeEach(() => {
+    jest.spyOn(console, 'error')
+    jest.spyOn(console, 'log')
+    jest.spyOn(console, 'warn')
+  })
+  afterEach(() => {
+    jest.resetModules()
+  })
+  afterAll(() => {
+    jest.clearAllMocks()
+  })
+
+  async function getInflectionSet(targetWord, languageID) {
+    const adapterTuftsRes = await ClientAdapters.morphology.tufts({
+      method: 'getHomonym',
+      params: {
+        languageID: languageID,
+        word: targetWord
+      }
+    })
+  
+    const testHomonym = adapterTuftsRes.result
+    const inflectionsViewSet = ViewSetFactory.create(testHomonym, locale)
+    return inflectionsViewSet
+  }
+
+  it('1 - checked Verb Paradigm1 - βουλεύῃς', async () => {
+    const inflectionsViewSet = await getInflectionSet('βουλεύῃς', Constants.LANG_GREEK)
+
+    expect(inflectionsViewSet.hasMatchingViews).toBeTruthy()
+
+    // console.info('inflectionsViewSet.matchingViews - ', inflectionsViewSet.matchingViews)
+    
+    expect(inflectionsViewSet.matchingViews.length).toEqual(1)
+    expect(inflectionsViewSet.matchingViews[0].constructor.name).toEqual('GreekVerbParadigmView')
+    expect(inflectionsViewSet.matchingViews[0].title).toEqual('ω-Verbs: Present System Active')
+    expect(inflectionsViewSet.matchingViews[0].paradigm).toBeDefined()
+    expect(inflectionsViewSet.matchingViews[0].paradigm.paradigmID).toEqual('verbpdgm1')
+    expect(inflectionsViewSet.matchingViews[0].paradigm.title).toEqual('ω-Verbs: Present System Active')
+    expect(inflectionsViewSet.matchingViews[0].linkedViews.length).toEqual(0)
+  })
+
+  it('2 - checked Verb Paradigm2 - βουλευέσθων', async () => {
+    const inflectionsViewSet = await getInflectionSet('βουλευέσθων', Constants.LANG_GREEK)
+
+    expect(inflectionsViewSet.hasMatchingViews).toBeTruthy()
+    
+    expect(inflectionsViewSet.matchingViews.length).toEqual(1)
+    expect(inflectionsViewSet.matchingViews[0].constructor.name).toEqual('GreekVerbParadigmView')
+    expect(inflectionsViewSet.matchingViews[0].title).toEqual('ω-Verbs: Present System Middle-Passive')
+    expect(inflectionsViewSet.matchingViews[0].paradigm).toBeDefined()
+    expect(inflectionsViewSet.matchingViews[0].paradigm.paradigmID).toEqual('verbpdgm2')
+    expect(inflectionsViewSet.matchingViews[0].paradigm.title).toEqual('ω-Verbs: Present System Middle-Passive')
+    expect(inflectionsViewSet.matchingViews[0].linkedViews.length).toEqual(0)
+  })
+
+  it('3 - no matches - checked Verb Paradigm3', async () => {
+   

```

Comment:
why is it blank? Is this table just not working?

Reply:
It means that I was not able to find the word.
For example here
I took the word from the paradigm-03.json
**βουλεύσει**
And it returns
Weak (1st) Aorist System Active (paradigm-08)
Future System (Active and Middle) with contraction in -έω (paradigm-04)

This way I checked all words from the json list.
And there is no match for paradigm-03

Reply:
Ok.  I think we should ask @monzug if she has found any words that (correctly) produce paradigm3

Reply:
yes, the ones that match this line in rules.csv such as βουλευθήσομαι
verbpdgm3	3	verb	aor_pass	passive		future

as said in issue #289 , match order is higher for verbpdgm4 table so this one is always generated